packages/jellyfish-transaction/src/script/dftx/dftx_governance.ts

```diff
@@ -99,3 +101,37 @@ export class CSetGovernance extends ComposableBuffer<SetGovernance> {
     ]
   }
 }
+
+export interface CreateProposal {
```

Comment:
Probably we can do this

```ts
interface Proposal { ... }
interface CreateCFP extends Proposal {
  type: 0x01
}
interface CreateVOC extends Proposal {
  type: 0x03
}
```

Reply:
Actually yeah, let's do this instead

Reply:
Okay let me adjust it.
What about acronyms ? Should they always be uppercased ?